easybuild/easyblocks/r/r.py

```diff
@@ -55,19 +58,39 @@ def prepare_for_extensions(self):
         self.cfg['exts_filter'] = EXTS_FILTER_R_PACKAGES
 
     def configure_step(self):
-        """Configuration step, we set FC, F77 is already set by EasyBuild to the right compiler,
-        FC is used for Fortan90"""
-        environment.setvar('FC', self.toolchain.get_variable('F90'))
+        """Custom configuration for R."""
+
+        # define $BLAS_LIBS to build R correctly against BLAS/LAPACK library
+        # $LAPACK_LIBS should *not* be specified since that may lead to using generic LAPACK
+        # see https://github.com/easybuilders/easybuild-easyconfigs/issues/1435
+        env.setvar('BLAS_LIBS', os.getenv('LIBBLAS_MT'))
+        self.cfg.update('configopts', "--with-blas --with-lapack")
 
         # make sure correct config script is used for Tcl/Tk
         for dep in ['Tcl', 'Tk']:
             root = get_software_root(dep)
             if root:
                 dep_config = os.path.join(root, 'lib', '%sConfig.sh' % dep.lower())
-                self.cfg.update('configopts', '-with-%s-config=%s' % (dep.lower(), dep_config))
+                self.cfg.update('configopts', '--with-%s-config=%s' % (dep.lower(), dep_config))
```

Comment:
Nasty typo here, this went undetected because the existing R easyconfigs already include `--with-tcl-config` and `--with-tk-config` in `configopts`...